html_sanitizer.rst

```diff
@@ -931,6 +931,76 @@ the HTML sanitizer: ``src``, ``href``, ``lowsrc``, ``background`` and ``ping``.
                 ->allowRelativeMedias()
         );
 
+Configure Max input length
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To prevent DoS attacks, the HTML sanitizer limits the input length to ``20000`` by default.
+Using this option, you can change the max input length.
+Inputs longer than this value will be truncated.
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/html_sanitizer.yaml
+        framework:
+            html_sanitizer:
+                sanitizers:
+                    app.post_sanitizer:
+                        # ...
+
+                        # specifies the max input length. Inputs longer than this value will be
+                        # truncated (default: 20000)
+                        max_input_length: 20000
```

Comment:
If 20000 is the default, we should configure 30k here.

the comment can be removed completely, instead I would suggest to write:

```suggestion
                        max_input_length: 30000 # default: 20000
```